### DISCUSSION TITLE
Karir sebagai cloud engineer AWS

### DISCUSSION THREAD

**Discussion by akriem** [awsugid/forum#31] [General]
Hallo semua, 

Mohon informasi, gambaran atau sarannya bagaimana prospek menjadi cloud engineer di AWS ya? Saat ini saya sedang belajar AWS dan rencana mw Abil sertifikasi CCP. 

Terima kasih 🙏

**Comment by rioastamal** [comment]
@akriem bacaan dari Werner Vogels berikut mungkin dapat memberikan gambaran: https://aws.amazon.com/executive-insights/content/werner-vogels-2023-tech-predictions-and-beyond/

**Reply by akriem** [reply]
Ok makasih om informasinya

**Comment by za** [comment]
Halo @akriem saya kira cukup prospektif karena migrasi infrastruktur ke cloud semakin lazim dilakukan belakangan ini.
